generic_tracers/cobalt_glbl.F90

```diff
@@ -0,0 +1,1531 @@
+!> COBALT_glbl module consists of core parameters and vars 
+!! to be used by generic COBALT related modules
+!<----------------------------------------------------------------
+module cobalt_glbl
+  use field_manager_mod, only: fm_string_len, fm_path_name_len      
+  implicit none
+  public  
+
+  !The following variables for using this module
+  ! are overwritten by generic_tracer_nml namelist
+  logical :: do_nh3_diag                             !< logic for setting NH3 diagnostic tracer field 
+
+! Namelist Options
+
+  character(len=10) ::  co2_calc = 'ocmip2'          !< carbonate formalation options. Other option is 'mocsy'
+  logical :: do_14c             = .false.            !< If true, then simulate radiocarbon 
+  logical :: do_nh3_atm_ocean_exchange = .false.     ! If true, then do NH3 air-sea exchange 
+  !
+  logical :: do_vertfill_pre = .false.             !< Returns tracer arrays with sensible values
+  logical :: debug              = .false.          !< not use   
+  real    :: o2_min_nit= 0.01e-6                   !< Oxygen threshold for nitrification (mol O2 kg-1)
```

Comment:
We could enter the values for o2_min_nit, k_o2_nit, irr_inhibit, gamma_nitrif and k_nh4_nitrif as we do the other model parameters as part of the namelist cleanup (i.e., in user_add_params).

Reply:
Agreed. I believe we should maintain a consistent approach for reading all the model input parameters. I will retain those parameters for now, and we should discuss our preference regarding how to handle COBALT parameter inputs.